sockets/index.js

```diff
@@ -19,15 +21,15 @@ module.exports = function (app, io) {
         ns.on('connection', function (socket) {
             if (clients() <= 1) {
                 object.onInit();
-                console.log(name, 'First connection');
+                logger.info(name, 'First connection');
             } else {
                 object.onConnect();
-                console.log(name, 'New connection');
+                logger.info(name, 'New connection');
```

Comment:
Not sure, maybe someone that is more familiar with that code can leave a comment? If I don't receive feedback I'll have a look myself when I find the time.